src/qt/languages/ru-RU.po

```diff
@@ -1402,7 +1402,7 @@ msgid "PS/2 Mouse"
 msgstr "Мышь PS/2"
 
 msgid "3M MicroTouch (Serial)"
-msgstr "3M MicroTouch (последовательный)"
+msgstr "3M MicroTouch (Последовательный)"
```

Comment:
No capitalization changes like these aren't needed, the translation doesn't need to follow English rules